monai/metrics/utils.py

```
@@ -465,7 +465,7 @@ def prepare_spacing(
 ENCODING_KERNEL = {2: [[8, 4], [2, 1]], 3: [[[128, 64], [32, 16]], [[8, 4], [2, 1]]]}
 
 
-@lru_cache(maxsize=None)
+@lru_cache(maxsize=None)  # noqa: UP033
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `UP033`)

Remove unused `noqa` directive

(RUF100)